dev/com.ibm.ws.openapi.ui/update-openapi-ui.sh

```diff
@@ -0,0 +1,77 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+#!/usr/bin/env bash
+
+# Check git username was provided as an argument
+if ! [ $# -eq 1 ]; then
+    echo "Missing git username argument."
+    exit 2
+else
+    git=$1
+fi
+
+# Ensure that a Node 16 environment is used
+if ! node -v | grep -q "^v16"; then
+    echo "A Node 16 environment must be used when running this script."
+    exit 1
+fi
+
+# Check version of npm is 8 or higher
+npm=`npm -v | sed 's/^[^0-9]*\([0-9]*\).*/\1/'`
+if [[ "${npm%.*}" -lt 8 ]]; then
+    echo "A version of npm 8 or higher must be used when running this script."
+fi
+
+# Installing script dependencies
+brew list jq >/dev/null 2>&1 || brew install jq
+brew list coreutils >/dev/null 2>&1 || brew install coreutils
+
+# Get Open Liberty remote alias from git
+remote=`git remote -v | grep "git@github.com:OpenLiberty/open-liberty.git (fetch)" | head -n1 | awk '{print $1;}'`
+
+# Checkout Open Liberty repo
+git fetch $remote
+git checkout integration
+git reset --hard $remote/integration
+git merge $remote/integration
+
+# Find new Swagger UI version
+export SWAGGER_UI_VERSION=$(jq -r '.packages."node_modules/swagger-ui".version' ./swagger-ui/package-lock.json)
+echo "Swagger UI version: $SWAGGER_UI_VERSION"
+
+# Update OpenAPI UI dependencies
+git branch "update-swagger-ui-$SWAGGER_UI_VERSION"
+git switch "update-swagger-ui-$SWAGGER_UI_VERSION"
+cd swagger-ui
+npx --yes npm-check-updates -u
+npm update
+
+# Copy SCSS from Swagger UI source
+rm -fr swagger-ui-src
+git clone --depth 1 -b v${SWAGGER_UI_VERSION} -c advice.detachedHead=false https://github.com/swagger-api/swagger-ui.git swagger-ui-src
+rm src/style/original/*
+rm -r src/style/core
+cp swagger-ui-src/src/style/* src/style/original
+shopt -s globstar
+cd swagger-ui-src/src
+gcp --parents core/**/*.scss ../../src/style
+cd ../..
+rm -rf swagger-ui-src
+
+# Build OpenAPI UI
+npm run build -- --mode=production
+
+# Commit and push
+git add .
+git commit -m "Update Swagger UI to $SWAGGER_UI_VERSION [auto]"
+git push --set-upstream "git@github.com:$git/open-liberty.git" "update-swagger-ui-$SWAGGER_UI_VERSION"
```

Comment:
Again, never do this for a script that a developer is supposed to run on their own workspace.